tests/Command/TestGhostCMSMigrator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Newspack\MigrationTools\Tests\Command;
+
+use Newspack\MigrationTools\Command\GhostCMSMigrator;
+use WP_UnitTestCase;
+
+/**
+ * Class TestGhostCMSMigrator
+ *
+ * @package newspack-migration-tools
+ */
+class TestGhostCMSMigrator extends WP_UnitTestCase {
+
+	/**
+	 * Test GhostCMS import CLI command
+	 */
+	public function test_cmd_ghostcms_import() {
```

Comment:
Would it be possible to test smaller units than the entire import?

The GhostHelper does most of the heavy lifting, I don't think GhostCMSMigrator needs a test at all. What do you think?